src/admin-views/base_admin_panel.php

```diff
@@ -1,35 +1,39 @@
+<?php
+	$total_tickets = Tribe__Tickets__Tickets_Handler::instance()->get_total_event_capacity( $post_id );
+	$container_class = 'tribe_sectionheader ticket_list_container';
+	$container_class .= ( empty( $total_tickets ) ) ? ' _blank' : '' ;
```

Comment:
1. The classname here needs a `tribe_` prefix to avoid naming collisions with other themes. It looks like we are using this to indicate that the event has `0` capacity. Maybe the classname could be `tribe_no_capacity`?

2. Would it make more sent to replace `empty( $total_tickets )` with `0 === $total_tickets`? When there is unlimited capacity, `$total_tickets` will be `-1`. Expressly checking for `0` might work better.